org.gtk.Gtk3theme.adw-gtk3.yaml

```diff
@@ -14,6 +14,7 @@ modules:
     build-commands:
       - install -d /usr/share/runtime/share/themes/adw-gtk3
       - cp -a adw-gtk3/gtk-3.0 /usr/share/runtime/share/themes/adw-gtk3
+      - cp -a adw-gtk3/gtk-4.0 /usr/share/runtime/share/themes/adw-gtk3
```

Comment:
As noted https://github.com/lassekongo83/adw-gtk3/issues/235#issuecomment-1649040839 This won't work.

Same for aw-gtk3 dark.